packages/web3-providers-http/test/integration/request.test.ts

```diff
@@ -4,15 +4,15 @@ import {
 	JsonRpcResponse,
 	JsonRpcResponseWithResult,
 } from 'web3-common';
-import { toWei, hexToNumber } from 'web3-utils';
 import HttpProvider from '../../src/index';
-import { accounts, clientUrl } from './config';
+// eslint-disable-next-line
```

Comment:
You should disable the specific rule that's being violated, not completely disable the linter